lib/src/exporter.dart

```diff
@@ -55,10 +55,48 @@ class Exporter {
         print('Skipped frame while enconding');
         continue;
       }
-      decodedImage.duration = frame.durationInMillis;
-      animation.addFrame(decodedImage);
+      decodedImage.frameDuration = frame.durationInMillis;
+      mainImage.frames.add(encodeGifWIthTransparency(decodedImage));
+    }
+
+    return image.encodeGif(mainImage);
+  }
+
+  static image.PaletteUint8 convertPalette(image.Palette palette) {
```

Comment:
This should probably be a private method.

Reply:
Done